tools/grpc/genproto.sh

```diff
@@ -1,9 +1,10 @@
-#!/usr/bin/env bash -x
```

Comment:
I removed the `-x` from here and moved it below because this script wouldn't run for me otherwise. I got this: 

```
$ ./tools/grpc/genproto.sh
/usr/bin/env: ‘bash -x’: No such file or directory
```

It is because the shebang line only allows a single argument after the executable. It interpreted `bash -x` as the argument to `env` and `env` couldn't find an executable by that name.